FE/src/components/TaskTagsList/styled-components.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export const StylesTaskTagsList = styled.div`
+    display:flex;
+    width: 170px;
+    overflow-x: scroll;
```

Comment:
Mais cuidado com a formataçao de codigo, aqui parece me que devia estar mais para tras mas posso estar errado 
```suggestion
 display:flex;
 width: 170px;
 overflow-x: scroll;
```

Reply:
Ai acho que ta certo o git é que faz parecer que tem uns espacos depois do `+`